test/integration/tasks.test.js

```diff
@@ -476,38 +476,6 @@ describe("Tasks", function () {
       expect(res).to.have.status(403);
       expect(res.body.message).to.be.equal("Status cannot be updated. Please contact admin.");
     });
-
-    it("should give a response message as 'Task updated but another task not found' if we have completed 100% task and a task is not available", async function () {
-      const taskData = tasksData[3];
-      taskId = (await tasks.updateTask(taskData)).taskId;
-
-      const dataToSend = { percentCompleted: 100 };
-
-      const res = await chai
-        .request(app)
-        .patch(`/tasks/self/${taskId}?dev=true`)
-        .set("cookie", `${cookieName}=${jwt}`)
-        .send(dataToSend);
-
-      expect(res).to.have.status(200);
-      expect(res.body.message).to.be.equal("Task updated but another task not found");
-    });
-
-    it("should give a response message 'task updated and another task got assigned' if we have completed the task 100% and another task is assigned to us", async function () {
-      const taskData = tasksData[4];
-      await tasks.updateTask(taskData);
-
-      const dataToSend = { percentCompleted: 100 };
-
-      const res = await chai
-        .request(app)
-        .patch(`/tasks/self/${taskId}?dev=true`)
-        .set("cookie", `${cookieName}=${jwt}`)
-        .send(dataToSend);
-
-      expect(res).to.have.a.status(200);
-      expect(res.body.message).to.be.equal("task updated and another task got assigned");
-    });
```

Comment:
this is removed as the logic for task assignment has changed, and we are not going to do it it on `/task/self/:id` endpoint